web/src/App.tsx

```diff
@@ -7,7 +7,7 @@ import { Button, Progress, Typography } from '@equinor/eds-core-react'
 import styled from 'styled-components'
 import { OpenAPI } from './api/generated'
 
-const hasAuthConfig = process.env.REACT_APP_AUTH === '1'
+const hasAuthConfig = import.meta.env.VITE_AUTH === '1'
```

Comment:
This is a difference between how webpack and vite works, because webpack allows files to contain _some_ node stuff, but Vite rather injects its own (e.g. import.meta.env) stuff. Potato/tomato, really..